core/ide/src/main/java/org/alice/ide/member/UnclaimedMethodsComposite.java

```diff
@@ -51,7 +54,18 @@
  * @author Dennis Cosgrove
  */
 public abstract class UnclaimedMethodsComposite extends FilteredMethodsSubComposite {
-  private final Comparator<AbstractMethod> comparator = FilteredMethodsSubComposite::compareMethodNames;
+  private final Comparator<AbstractMethod> comparator = UnclaimedMethodsComposite::compareMethodNames;
+
+  protected static int compareMethodNames(AbstractMethod methodA, AbstractMethod methodB) {
+    if (methodA == null) {
```

Comment:
I mostly just want to share what I recently found, but these null checks can be dropped if you define the comparator above as `comparator = Comparator.nullsLast(UnclaimedMethodsComposite::compareMethodNames)`.
This is a simplification we could apply elsewhere.